src/components/pages/Profile/Profile.js

```diff
@@ -34,10 +36,19 @@ class Profile extends React.Component {
   }
 
   petFormSubmitEvent = (pet) => {
-    petRequests.createPet(pet);
-    this.getPartedPets();
+    const { isEditingPet, petIdToEdit } = this.state;
+    if (isEditingPet) {
+      petRequests.editPet(petIdToEdit, pet);
```

Comment:
When I edited my pet the card did not update until I hit refresh.  I think if you turn the `editPet` method in the `petRequests.js` file into a Promise and wrap the `getPartedPets` call inside the then from the editPet it will update

Reply:
Refreshing now 👍